.drone.star

```diff
@@ -1609,6 +1618,62 @@ def notify(ctx):
         },
     }
 
+def ocisServerWithIdp():
+    environment = {
+        "GRAPH_IDENTITY_BACKEND": "ldap",
+        "GRAPH_LDAP_SERVER_WRITE_ENABLED": "true",
+        "LDAP_URI": "ldaps://0.0.0.0:9235",
+        "LDAP_INSECURE": "true",
+        "LDAP_BIND_DN": "uid=libregraph,ou=sysusers,o=libregraph-idm",
+        "LDAP_BIND_PASSWORD": "idm",
+        "LDAP_USER_BASE_DN": "ou=users,o=libregraph-idm",
+        "LDAP_USER_SCHEMA_ID": "ownclouduuid",
+        "LDAP_USER_SCHEMA_MAIL": "mail",
+        "LDAP_USER_SCHEMA_USERNAME": "uid",
+        "LDAP_USER_OBJECTCLASS": "inetOrgPerson",
+        "LDAP_GROUP_BASE_DN": "ou=groups,o=libregraph-idm",
+        "LDAP_GROUP_SCHEMA_ID": "ownclouduuid",
+        "LDAP_GROUP_SCHEMA_MAIL": "mail",
+        "LDAP_GROUP_SCHEMA_GROUPNAME": "cn",
+        "LDAP_GROUP_SCHEMA_MEMBER": "member",
+        "LDAP_GROUP_OBJECTCLASS": "groupOfNames",
+        "IDP_INSECURE": "true",
+        "IDP_LDAP_BIND_DN": "uid=idp,ou=sysusers,o=libregraph-idm",
+        "IDP_LDAP_BIND_PASSWORD": "idp",
+        "IDP_LDAP_BASE_DN": "ou=users,o=libregraph-idm",
+        "IDP_LDAP_LOGIN_ATTRIBUTE": "uid",
+        "PROXY_ACCOUNT_BACKEND_TYPE": "cs3",
+        "PROXY_ENABLE_BASIC_AUTH": "true",
+        "STORAGE_LDAP_BIND_DN": "uid=reva,ou=sysusers,o=libregraph-idm",
+        "STORAGE_LDAP_BIND_PASSWORD": "reva",
+        "OCS_ACCOUNT_BACKEND_TYPE": "cs3",
+        "OCIS_RUN_EXTENSIONS": "settings,storage-metadata,graph,graph-explorer,ocs,store,thumbnails,web,webdav,storage-frontend,storage-gateway,storage-userprovider,storage-groupprovider,storage-authbasic,storage-authbearer,storage-authmachine,storage-users,storage-shares,storage-public-link,storage-appprovider,storage-sharing,proxy,idp,nats,idm,ocdav",
+        "OCIS_LOG_LEVEL": "error",
+        "OCIS_INSECURE": "true",
+        "OCIS_URL": "https://ocis-server:9200",
```

Comment:
the reason for the TUS tests failing was due to not having `OCIS_URL` pointed to `https://ocis-server:9200`

Reply:
Now the CI is green: https://drone.owncloud.com/owncloud/ocis/10789